.github/workflows/base-branch-check.yml

```diff
@@ -0,0 +1,25 @@
+name: Base Branch Checker
+
+on:
+  pull_request:
+    branches-ignore:
+      - development
```

Comment:
Can this only include the `master` branch? PRs to other branches should be allowed without any warnings.

Reply:
I applied the change.